apps/dashboard/src/app/applications.ts

```diff
@@ -153,21 +153,21 @@ export const applications: IApplications = <const>{
         classDimension: 2,
         counterfactualData: [adultCounterfactualData],
         dataset: adultCensusWithFairnessDataset,
-        errorAnalysisConfig: [adultCensusCausalErrorAnalysisConfig],
+        errorAnalysisData: [adultCensusCausalErrorAnalysisData],
         modelExplanationData: [adultCensusWithFairnessModelExplanationData]
       } as IModelAssessmentDataSet,
       adultCensusIncomeNoCausalData: {
         classDimension: 2,
         counterfactualData: [adultCounterfactualData],
         dataset: adultCensusWithFairnessDataset,
-        errorAnalysisConfig: [adultCensusCausalErrorAnalysisConfig],
+        errorAnalysisData: [adultCensusCausalErrorAnalysisData],
         modelExplanationData: [adultCensusWithFairnessModelExplanationData]
       } as IModelAssessmentDataSet,
       adultCensusIncomeNoCounterfactualData: {
         causalAnalysisData: [adultCensusCausalAnalysisData],
         classDimension: 2,
         dataset: adultCensusWithFairnessDataset,
-        errorAnalysisConfig: [adultCensusCausalErrorAnalysisConfig],
+        errorAnalysisData: [adultCensusCausalErrorAnalysisData],
         modelExplanationData: [adultCensusWithFairnessModelExplanationData]
       } as IModelAssessmentDataSet,
       adultCensusIncomeNoModelData: {
```

Comment:
This renders WITH model data when I click through the UI in the blob. Something is off here.

Reply:
I am confused